.kokoro/osx.sh

```diff
@@ -24,15 +24,15 @@ command curl -sSL https://rvm.io/mpapis.asc | gpg --import -
 command curl -sSL https://rvm.io/pkuczynski.asc | gpg --import -
 rvm get head --auto-dotfiles
 
-versions=(2.4.10 2.5.8 2.6.6 2.7.2)
+versions=(2.4.10 2.5.8 2.6.6 2.7.2 3.0.0)
```

Comment:
Sorry about this. Would you be able to revert all changes to this file? (Both this line and the change further down.) We have some issues right now in our CI setup (unrelated to the new library creation process)